src/Command/SimpleCommandFactory.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Equip\Queue\Command;
+
+use Equip\Command\CommandInterface;
+use Equip\Queue\Exception\CommandException;
+use Equip\Queue\Exception\HandlerException;
+
+class SimpleCommandFactory implements CommandFactoryInterface
+{
+    public function make($command)
+    {
+        if (!class_exists($command)) {
+            throw CommandException::notFound($command);
+        }
+
+        $command = new $command;
```

Comment:
Can we add an implementation that uses `auryn/injector` and include it in the `suggest` section of `composer.json` ?